change/react-native-windows-2020-08-25-14-42-13-archDeploy-0.62.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "[0.62] Fix deploy bug when framework packages are installed but not for all archs",
```

Comment:
FYI, the [0.62] part will show up in changelogs, which we don't  necessarily want. Consider leaving the version number out in future PRs. You can add free-form text in "yarn change" instead of picking a commit title if you want the changelog entry to be custom.

Reply:
See https://github.com/microsoft/react-native-windows/releases for an example of what the automated release notes end up looking like.